src/utils/utils.ts

```diff
@@ -159,22 +164,39 @@ export async function getResultsJson() {
   return { results: [] };
 }
 
-export function readResultsFromFile(resultJsonPath: string, scan: string): CxResult[] {
-  let results = undefined;
-  if (fs.existsSync(resultJsonPath) && scan) {
-    const jsonResults = JSON.parse(
-      fs
-        .readFileSync(resultJsonPath, "utf-8")
-        .replace(/:([0-9]{15,}),/g, ':"$1",')
-    );
-    if (jsonResults.results) {
-      results = orderResults(jsonResults.results);
-    }
-    else {
-      results = [];
+
+export function readResultsFromFile(resultJsonPath: string, scan: string): Promise<CxResult[]> {
```

Comment:
does the new library throw any exception? if yes - we should add try-catch to this logic

Reply:
this is the err handling:
.on('error', (error) => {
          reject(error);
        });

Reply:
do you think we should log the error?